infrastructure/quick-deploy/aws/all/variables.tf

```diff
@@ -677,3 +677,70 @@ variable "image_tags" {
   description = "Tags of images used"
   type        = map(string)
 }
+
+# The name of the secrets.
+variable "fluent_bit_secret_name" {
+  description = "the name of the fluent-bit secret"
+  type        = string
+  default     = "fluent-bit"
+}
+
+variable "grafana_secret_name" {
+  description = "the name of the grafana secret"
+  type        = string
+  default     = "grafana"
+}
+
+variable "prometheus_secret_name" {
+  description = "the name of the prometheus secret"
+  type        = string
+  default     = "prometheus"
+}
+
+variable "metrics_exporter_secret_name" {
+  description = "the name of the metrics exporter secret"
+  type        = string
+  default     = "metrics-exporter"
+}
+
+variable "partition_metrics_exporter_secret_name" {
+  description = "the name of the partition metrics exporter secret"
+  type        = string
+  default     = "partition-metrics-exporter"
+}
+
+variable "seq_secret_name" {
+  description = "the name of the seq secret"
+  type        = string
+  default     = "seq"
+}
+
+variable "shared_storage_secret_name" {
+  description = "the name of the shared-storage secret"
+  type        = string
+  default     = "shared-storage"
+}
+
+variable "deployed_object_storage_secret_name" {
+  description = "the name of the deployed-object-storage secret"
+  type        = string
+  default     = "deployed-object-storage"
+}
+
+variable "deployed_table_storage_secret_name" {
+  description = "the name of the deployed-table-storage secret"
+  type        = string
+  default     = "deployed-table-storage"
+}
+
+variable "deployed_queue_storage_secret_name" {
+  description = "the name of the deployed-queue-storage secret"
+  type        = string
+  default     = "deployed-queue-storage"
+}
+
+variable "s3_secret_name" {
+  description = "the name of the S3 secret"
+  type        = string
+  default     = "s3"
+}
```

Comment:
There is no need to variabilize this at the root module level

Reply:
indeed I was thinking the same and Lilia will be happy :)